desktop/libs/notebook/src/notebook/connectors/trino.py

```diff
@@ -47,7 +45,7 @@ def decorator(*args, **kwargs):
     except RestException as e:
       try:
         message = force_unicode(json.loads(e.message)['errors'])
-      except:
+      except Exception as e:
```

Comment:
Should this change the character to not shadowed by the previous exception, like `Exception as ex`?

Reply:
done